### DISCUSSION TITLE
API stability

### DISCUSSION THREAD

**Discussion by ejannett** [jobrunr/jobrunr#435] [Q&A] [answered]
Hi,

I am using JobRunR free edition. I would like to know the status of the APIs . Which one listed in the examples
and the current repository are considered public/stable ? 
I am especially interested of knowing if org.jobrunr.storage.StorageProvider is a public/stable API  ? 

Thank in advance

regards

**Comment by rdehuyss** [comment, marked as answer]
StorageProvider is safe/public to use. A change in that class results in a major version bump of JobRunr.

**Reply by rdehuyss** [reply]
fyi - I also reached out to you via mail.
